src/Directory.Build.targets

```diff
@@ -0,0 +1,34 @@
+<Project>
```

Comment:
Why do we need both this file _and_ Directory.Build.props? Shouldn't it just be one or the other?

Reply:
I was troubleshooting an issue which made me to add that - we don't need that any more, I think! I will update!